examples/seismic/model.py

```diff
@@ -377,14 +405,22 @@ def __init__(self, origin, spacing, shape, space_order, vp, nbpml=20,
         else:
             self.theta = 0
 
-        if phi is not None:
+        if phi is not None and self.grid.dim > 2:
```

Comment:
should we raise an error/warning if we get down to this place but ``self.grid.dim`` is <= 2?

Reply:
Probably not necessary no that I am thinking of it. I probably only need to modify the `_physical_parameter`.

Reply:
reminder: we can drop this, then?

Reply:
Not quite, still needs to raise a warning and I think this is the best place to do